src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.module.scss

```diff
@@ -89,6 +88,24 @@ $laptop-range-small-end: 929px;
   z-index: $z-index-navbar;
 }
 
+:global(.isScrolled) {
+  .laptopAndUpAnimation {
+    background-color: var(--White);
+    color: var(--GrayPrimary--translucent);
+    transition: background-color 1s ease-in-out;
+  }
+}
+
+:global(.isUnscrolled) {
+  .laptopAndUpAnimation {
+    transition: background-color 1s ease-in-out;
+  }
+}
+
+.laptopAndUpAnimation {
+  border-bottom: none;
+}
```

Comment:
@hangngo21 it's a little unclear in the mocks, but I think the border should be animated, same as the background color, so that it fades in when `.isScrolled` is true and be hidden when `.isUnscrolled` is true.